user_templates/mcu_defaults/main/Mellow_Fly_Gemini_v3.cfg

```diff
@@ -0,0 +1,51 @@
+
+#------------------------------------------#
+#### CUSTOM TEMPLATE MCU definition ########
```

Comment:
Please update the header with the correct MCU name (Mellow Fly Gemini v3)

Reply:
Someone should do a similar review for the Fly-Super8, since that was what I copied as a base 😅

Reply:
Thanks for the hint, I'll fix this!